vgen/src/main/java/io/gapi/vgen/go/GoLanguageProvider.java

```diff
@@ -50,7 +50,7 @@
 
   @SuppressWarnings("unchecked")
   public <Element> GeneratedResult generate(
-      Element element, SnippetDescriptor snippetDescriptor, GoGapicContext context) {
+      Element element, SnippetDescriptor snippetDescriptor, Object context) {
```

Comment:
Do DiscoveryContext and GoGapicContext  have a more specific common ancestor than Object? If so, let's use that.